safe-room.lic

```diff
@@ -130,6 +134,23 @@ class SafeRoom
     bput("put my material in my #{settings.thanatology['harvest_container']}", 'You put', 'What were') if right_hand
   end
 
+  def use_divine_charm
```

Comment:
Since this returns a bool, the convention is to name it with `?` at the end, i.e. `use_divine_charm?`